src/Components/Web.JS/src/Platform/BootConfig.ts

```diff
@@ -1,13 +1,17 @@
+import { WebAssemblyBootResourceType } from "./WebAssemblyStartOptions";
```

Comment:
Nitpick, please ignore if you want. I know the tooling will have done this for you.

```suggestion
import { WebAssemblyBootResourceType } from './WebAssemblyStartOptions';
```